hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/statistics/IOStatisticsLogging.java

```diff
@@ -29,8 +30,12 @@
 import org.apache.hadoop.classification.InterfaceAudience;
 import org.apache.hadoop.classification.InterfaceStability;
 import org.apache.hadoop.fs.statistics.impl.IOStatisticsBinding;
+import org.apache.log4j.Level;
```

Comment:
needs to use SLF4J. Is this an accidental import?

Reply:
Accidentally added it, removed it in the latest commit.